src/installer/tests/Directory.Build.props

```diff
@@ -4,12 +4,11 @@
   <PropertyGroup>
     <TestArtifactsOutputRoot>$(ArtifactsDir)tests\host\$(TargetOS).$(TargetArchitecture).$(Configuration)\</TestArtifactsOutputRoot>
     <TestInfraTargetFramework>$(NetCoreAppToolCurrent)</TestInfraTargetFramework>
-    <TestCaseFilter>category!=failing</TestCaseFilter>
-    <TestRunnerAdditionalArguments>--filter $(TestCaseFilter) -v detailed</TestRunnerAdditionalArguments>
+    <TestCaseFilter>/[category!=failing]</TestCaseFilter>
```

Comment:
TestCaseFilter naming is usually tied to VSTest. That's not what you run with now. This is now xunit's filter query.